_life/money-americans.md

```diff
@@ -272,6 +276,38 @@ Vaud is known to unofficially grant automatic extensions until June. Two, that t
 cantons collect the federal, cantonal, and communal taxes in Switzerland, so you only
 have to file one return. Good luck!
 
+## Internships in the US for non-citizens
+
+If there is any possibility you might live in the US in the future, open an account
+at a US bank soon after you arrive for your internship. Many banks offer basic
+checking accounts with _zero fees_ if you meet some relatively easy criteria, so
+don't pay unless you really have to. Some banks will require you to have a social
+security number, but if you are being paid for your internship, your employer should
```

Comment:
FWIW, at least as of 2017 First Tech didn't require an SSN to open an account but did to have online banking